### PR TITLE
fix: SDK credential creation

### PR DESCRIPTION
**Pull request by juliosantos**
@pkoch pretty sure this is why Bob couldn't decrypt the credential Alice created for him yesterday

### CODE REVIEW

**Comment by pkoch** [comment]
Huh. That would make sense, yeah. However, we did see the right value on the DB. So, I'm still unconvinced this would fix it.

**Comment by juliosantos** [comment]
You're right — I missed a ton.

File: packages/idos-sdk-js/src/lib/enclave-providers/iframe-enclave.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 export class IframeEnclave {
-  hostUrl = new URL("https://foo.com:5174");
+  hostUrl = new URL(import.meta.env.VITE_IDOS_ENCLAVE_URL);
```
Comment (juliosantos) [inline packages/idos-sdk-js/src/lib/enclave-providers/iframe-enclave.js:2]: ha, thanks @ditoglez 🙏

**Review by pkoch — APPROVED** [review]
I see nothing dumb. I'd have to click it a bunch to give you stronger guarantees. Let me know if you'd rather have me do that. Otherwise, ship.

**Comment by pkoch** [comment]
Spoke with Júlio on a call, taking this over.

**Pull request merged**